Source/Client/Sound/SoundSystem.cs

```diff
@@ -79,24 +81,30 @@ public static bool Initialize(Form target)
 
         // Get settings from configuration
         playeffects = General.config.ReadSetting("sounds", true);
-        effectsvolume = CalcVolumeScale((float)General.config.ReadSetting("soundsvolume", 100) / 100f);
+        playmusic = General.config.ReadSetting("music", true);
+        effectsvolume = CalcVolumeScale(General.config.ReadSetting("soundsvolume", 100) / 100f);
+        musicvolume = CalcVolumeScale(General.config.ReadSetting("musicvolume", 100) / 100f);
```

Comment:
In the old code, the default music volume was 50. Let's make it so here as well?